kibana/meta/collectd.yml

```diff
@@ -0,0 +1,11 @@
+{%- if pillar.kibana.server is defined %}
+{%- from "kibana/map.jinja" import server with context %}
+
+{%- if server.enabled %}
```

Comment:
more robust and coherent w/ other states

{%- if server.get('enabled', False) %}